extensions/Cake.BuildSystems.Module.yml

```diff
@@ -10,9 +10,7 @@ Description: This Cake module will introduce a number of features for running in
 Categories:
 - ci
 TargetCakeVersion: 
-TargetFrameworks:
-- net461
-- netstandard2.0
-AnalyzedPackageVersion: 2.0.1
+TargetFrameworks: []
```

Comment:
Why did we loose target frameworks here

/cc @Jericho

Reply:
Evidently the organization of the nuget package has changed. Assemblies are located under the `lib` folder rather than being organized in the `lib\netstandard2` folder as expected:

![image](https://user-images.githubusercontent.com/112710/108577642-bc7d2380-72ef-11eb-889a-823539fd838b.png)

The consequence of file being organized this way is that when we invoke 
```csharp
var frameworks = package
    .GetLibItems()
    .Select(i => i.TargetFramework.GetShortFolderName())
    .Except(new[] { "any" })
    .ToArray();
```

we get an empty array.

Reply:
To be fixed by https://github.com/cake-contrib/Cake.BuildSystems.Module/pull/56

Reply:
I've just released v3.0.1 which fixes this. I'll close this PR and issue, so that the AddinDisco can create a new one later tonight.